GenderPayGap.WebUI/Controllers/Admin/AdminDownloadsController.cs

```diff
@@ -458,6 +459,19 @@ public FileContentResult EhrcAllOrganisationsForYear_AdminPage(int year)
             return GenerateEhrcAllOrganisationsForYearFile(dataRepository, year);
         }
 
+        [HttpGet("downloads/organisations-without-submitted-returns")]
+        public FileContentResult DownloadOrganisationsWithNoSubmittedReturns(int year)
+        {
+            IEnumerable<Organisation> organisationsWithNoSubmittedReturns = GetOrganisationsWithNoSubmittedReturns(year);
+
+            IEnumerable<dynamic> records = BuildOrganisationsWithNoSubmittedReturnsRecords(organisationsWithNoSubmittedReturns);
+
+            string fileDownloadName = $"Gpg-NoSubmissionsFor{year}-{VirtualDateTime.Now:yyyy-MM-dd HH:mm}.csv";
```

Comment:
Does the GDS Formatter have this date format on it? Might be worth adding it there and using it here